pyproject.toml

```diff
@@ -8,8 +8,9 @@ readme = "README.md"
 packages = [{include = "ubireader"}]
 
 [tool.poetry.dependencies]
-python = ">=3.9"
+python = ">=3.10"
```

Comment:
this is a big change, I don't think we should drop support for Python 3.9 since it will be only EoL in October 2025

Reply:
cryptography lib is needing 3.9.2 at least would it be a solution to use this version at least? Or should we keep it with 3.9? Otherwise we need to switch to cryptography==39.0.0 if I got things right.

Reply:
switched to python 3.9.2. this is working with pyca.cryptography